# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/cocoa-foundation/src/base.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/cocoa-foundation/src/base.rs b/third_party/rust/cocoa-foundation/src/base.rs
deleted file mode 100644
--- a/third_party/rust/cocoa-foundation/src/base.rs
+++ /dev/null
@@ -1,28 +0,0 @@
-// Copyright 2013 The Servo Project Developers. See the COPYRIGHT
-// file at the top-level directory of this distribution.
-//
-// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
-// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
-// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
-// option. This file may not be copied, modified, or distributed
-// except according to those terms.
-
-use objc::runtime;
-
-pub use objc::runtime::{BOOL, NO, YES};
-
-pub type Class = *mut runtime::Class;
-#[allow(non_camel_case_types)]
-pub type id = *mut runtime::Object;
-pub type SEL = runtime::Sel;
-
-#[allow(non_upper_case_globals)]
-pub const nil: id = 0 as id;
-#[allow(non_upper_case_globals)]
-pub const Nil: Class = 0 as Class;
-
-/// A convenience method to convert the name of a selector to the selector object.
-#[inline]
-pub fn selector(name: &str) -> SEL {
-    runtime::Sel::register(name)
-}